lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -16,10 +17,9 @@ def initialize(serializer, options = {})
         end
 
         def serializable_hash(options = nil)
-          options = {}
```

Comment:
Am curious if you moved the definition of options here to the two places below for style reasons or if something was broken.